docs/gds-supported-platform/getting-started-gsp-local.md

```diff
@@ -1,54 +1,106 @@
-# Getting started with a local GDS Supported Platform development cluster
+# Set up a local GDS Supported Platform instance
 
-These instructions tell you how to set up a local GDS Supported Platform (GSP) cluster and host an app on that cluster.
+These instructions tell you how to set up a local instance of the GDS Supported Platform (GSP) and run the example `govuk-prototype-kit` app on that instance. This process is useful for testing your app before deploying that app to a production environment. You will:
 
-This process is resource-intensive and you must set the resource amount used by Docker to the following:
-- 4 CPUs
-- 8 gigabytes of memory
+- install prerequisite software
```

Comment:
can we link this table of contents to the corresponding h2 tags for convenience?

Reply:
yes